src/Utils/obiDialogParser.ts

```diff
@@ -181,39 +207,89 @@ export class ObiDialogParser {
         }
         return trainDialogs
     }
-    
-    private async getScorerStepfromActivity(prompt: string): Promise<CLM.TrainScorerStep> {
-    
+
+    private async getScorerStepFromActivity(prompt: string): Promise<CLM.TrainScorerStep> {
         const parsedActivity = prompt.substring(prompt.indexOf("[") + 1, prompt.lastIndexOf("]")).trim()
         let response = this.composerDialog.lgMap.get(parsedActivity)
         if (!response) {
             // LARS thow error once CCI .dialog transformer has been fixed
-            response = { text: "Can't Parse LG", suggestions: []}
-           //throw new Error(`LG name ${prompt} undefined`)
+            response = { text: "Can't Parse LG", suggestions: [] }
```

Comment:
Would be good to follow up with Swadheen to understand why we are still getting bad data